src/styles.scss

```diff
@@ -139,15 +139,15 @@ $border-grey: #E5E5E5;
 $staking-grey: #F0F0F0;
 $light-grey-for-divider: #F8F8F8;
 $light-grey-for-divider-2: #FAFAFA;
-$buy-button-blue: #0058F7;
```

Comment:
let's use this hex value as the button color for light, dark, and icy dark themes.

Reply:
Ok sure